### PR TITLE
add in some serial setup tests; a little make cleanup

### PR DESCRIPTION
**Pull request by mikebrow**
A follow up on https://github.com/containerd/go-cni/pull/99 to add in some serial setup buckets..

Also made a couple nits to the makefile..

Signed-off-by: Mike Brown <brownwm@us.ibm.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /bin/
 coverage.txt
 profile.out
+.vscode
```
Comment (fuweid) [inline .gitignore:4]: oops. I think we should remove this line~

Comment (mikebrow) [reply .gitignore]: sure .. removed

never sure of when it's ok to add .gitignore paths / files   just removing an eye sore from my git status check ...

I'll figure out why vsc started adding those maybe I can config it out vs ignore it :-)

**Comment by fuweid** [comment]
ping @estesp @AkihiroSuda @dmcgowan

**Review by estesp — APPROVED** [review]
LGTM

**Pull request merged**